commcare_connect/events/models..py

```diff
@@ -0,0 +1,151 @@
+from django.db import models
+
+from commcare_connect.users.models import User
+
+
+class Event(models.Model):
+
+	class Type(models.TextChoices):
+	    INVITE_SENT = "is", gettext("Invite Sent")
```

Comment:
I think the slugs should be human readable. No reason to save a few bytes, and it will make it easier to use them later

Reply:
I think since we are likely to be analyzing these events using tools like superset, having readable ones outside of python code could be really useful, and im not sure the few bytes are going to make a big difference even in the long run